### PR TITLE
Split build image actions

### PR DESCRIPTION
**Pull request by shleeable**

### CODE REVIEW

**Comment by shleeable** [comment]
This is good to merge.

File: .github/workflows/build-image.yml [hunk 1]
```diff
@@ -11,7 +11,36 @@ on:
       - .github/workflows/build-image.yml
       - Dockerfile
 jobs:
-  build-image:
+  build-image-amd64:
```
Comment (ykzts) [inline .github/workflows/build-image.yml:14]: I think it's smart to use build matrix.

see https://docs.github.com/en/actions/using-workflows/advanced-workflow-features#using-a-build-matrix

Comment (shleeable) [reply .github/workflows/build-image.yml:14]: My main problem is the logging for both builds being piped to the same log file... so it jumps backforward and forwards between builds.

If matrix solves THIS problem. I'm good.

Comment (shleeable) [reply .github/workflows/build-image.yml:14]: image

Comment (ykzts) [reply .github/workflows/build-image.yml:14]: They can be placed in the same group, but will be displayed separately.

**Comment by shleeable** [comment]
@ykzts updated using the matrix method

**Review by ClearlyClaire — APPROVED** [review]

**Review by ykzts — APPROVED** [review]

**Comment by Gargron** [comment]
Breaks Docker tagging, please resubmit once that is figured out.

**Pull request merged**